### ISSUE TITLE
Make "Using facets and filters" dismissible

### ISSUE THREAD

**Issue by hughlilly** [OpenRefine/OpenRefine#4539]
I'd like to be able to dismiss the "Using facets and filters" modal, which appears in the sidebar.

### Proposed solution
This would be a per-project setting; if a user dismisses the message for one project and starts a new one, the message would reappear on the new project. A small "x" in the left corner would suffice as an interface element, in my opinion.

### Alternatives considered
I realise the sidebar can be collapsed totally, but I don't think there are any alternatives to this specific problem.

### Additional context
Screenshot:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/12046008/154405680-f10d989a-3f5d-4de6-9d85-a91d9948009c.png">

Version 3.5.2 [e3efd4e] on Safari Version 15.2 (17612.3.6.1.6) on macOS 12.1 (21C52).

Thanks,
Hugh

**Comment by thadguidry** [comment]
If I understand this correctly, you want a small X (close) in the upper right , to the top right of the blue diamond that would remove the text message square?
What kind of problem is it causing you? Curious.

**Comment by hughlilly** [comment]
Yes, that's (mostly) correct—my preference for the placement of the 'x' would be the upper left corner.

It's purely a visual distraction, that's all. It would be nice if I could dismiss it so as to more easily concentrate on using the product. I see it as part of a "tour" of how the software works, but now that I'm familiar with it, I don't need to see this kind of element/message over and over.

**Comment by Babi-B** [comment]
A similar modal appears for the "Undo/Redo" option too

![image](https://user-images.githubusercontent.com/66228179/162858275-b19233b2-a639-4d89-9b69-e80bf7ae33a2.png)

**Comment by Babi-B** [comment]
>This would be a per-project setting; if a user dismisses the message for one project and starts a new one, the message would reappear on the new project.

So from this statement, the "x" button will just add a `.hide` method to the modal

**Comment by antoine2711** [comment]
> So from this statement, the "x" button removes the visibility of the modal in that instance?

Yes @Babi-B.

Also, there are 2 of them, one in the facet panel and another in the history panel.

I think the top-right corner would be more like other dialogs. 
What could be done also is a preference that would always hide them.

Regards, Antoine

**Comment by antoine2711** [comment]
@hughlilly: how would you feel about a preference on your machine that would prevent those « splash » to display?
Something like `ui.browsing.facetsHideSplash`?

Regards, Antoine

**Comment by hughlilly** [comment]
> @hughlilly: how would you feel about a preference on your machine that would prevent those « splash » to display? Something like `ui.browsing.facetsHideSplash = true`?
> 
> Regards, Antoine

Hello; apologies for my very late reply. That would be great — but as others have mentioned it would also be good to have an "x" in the corner to close the dialog, either for that session or permanently. Would it be going to far to have `ui.browsing.facetsHideSplash` set to `true` by pressing the 'x'?

Hugh

**Comment by antoine2711** [comment]
@hughlilly: 
« Would it be going to far to have `ui.browsing.facetsHideSplash` set to `true` by pressing the 'x'? »


What is being coded is 2 things: a close button, for the session, and a checkbox that sets the preference. So if you check the box, and then close the splash, you will not see it again, for any projects, unless you revert the preference.

Regards, Antoine
